ops/deploy.sh

```diff
@@ -131,6 +131,17 @@ kubectl --namespace=data-repo apply -f "${SCRATCH}/oidc-proxy-deployment.yaml"
 kubectl --namespace=data-repo apply -f "${SCRATCH}/cloudsql-proxy.yaml"
 kubectl --namespace=data-repo apply -f "${SCRATCH}/oidc-ingress.yaml"
```

Comment:
`export KUBE_NAMESPACE=${KUBE_NAMESPACE:-'data-repo'}`

Could find and replace all of the following in the deploy.sh

```--namespace=data-repo```
to 
```--namespace=${KUBE_NAMESPACE}```